nextjs-logs-manager/lib/format-date.ts

```
@@ -0,0 +1,50 @@
+  options: DateFormatOptions = {},
+): string => {
+  const {
+    locale = "en-US",
+    format = "medium",
+    includeTime = true,
+    hour12 = true,
+  } = options;
+
+  const dateObj = date instanceof Date ? date : new Date(date);
+
+  if (isNaN(dateObj.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)